app/views/layouts/provider/login.html.slim

```diff
@@ -7,7 +7,7 @@ html lang="en"
 
     = render 'provider/analytics'
     = stylesheet_link_tag 'loginPageFonts'
-    = javascript_pack_tag 'pf4BaseStyles'
+    = javascript_pack_tag 'loginPageStyles'
```

Comment:
@didierofrivia I've found that `using stylesheet_pack_tag to point at a CSS file is discouraged, as "packs" should only be JS files` ([here](https://github.com/rails/webpacker/issues/1575))
I think we can use the js file as entry point, and from there import the css files needed